### ISSUE TITLE
Add a reset button

### ISSUE THREAD

**Issue by ianhi** [QuantStack/ipycytoscape#71]
Similar to the home button in ipympl:

![image](https://user-images.githubusercontent.com/10111092/82166694-4393fb80-9887-11ea-917b-c6b0e79a35f5.png)

When I'm scrolling up an down the notebook I can end up accidently zooming way out and end up being unable to find the graph again.  This would give me a way to return. Ideally if there was a button there would be an option to hide it.

**Comment by marimeireles** [comment]
That's cool, it'd be cool to have a menu like this I think.
I think you can also rerun the same cell and get the same result, right?

**Comment by ianhi** [comment]
Yeah you can but that will also relayout the graph. Better would be to expose https://js.cytoscape.org/#cy.reset A pretty good fix would just to be to add that method to the python object. i.e. python would have

```python
def reset():
    self.send({'name': 'reset'}, [])
```

and in typescript the model adds

```typescript
this.on('msg:custom', this._handleMessage.bind(this));
....

private _handleMessage(command: any, buffers: any) {
    switch(command.name){
....
        case 'reset: {
            this._forEachView((view) => view.reset())
        }
    }
}
```
and the views would implement the `reset` method. Then if we also wanted a button the button could just send the same message.

**Comment by ianhi** [comment]
The infrastructure for this is being worked on in https://github.com/QuantStack/ipycytoscape/pull/109
